<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Voronoi Illustration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="h_0" stretch="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackwgt_0l">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_voronoi">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="GL2DWidget" name="oglwgt_voronoi"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3d">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QOpenGLWidget" name="oglwgt_3d"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="v0r">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_point_generation">
          <property name="text">
           <string>Point Generation</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="h_0r1u">
          <item>
           <widget class="QSpinBox" name="spbox_random">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbtn_random">
            <property name="text">
             <string>Random</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pbtn_manual">
            <property name="text">
             <string>Manual</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbtn_file">
            <property name="text">
             <string>File</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="hl_0r1u">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="vsp_0r">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pbtn_voronoi">
            <property name="text">
             <string>Voronoi</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbtn_illustration">
            <property name="text">
             <string>Illustration</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GL2DWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>gl2Dwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
